administrator/components/com_fields/src/Model/FieldsModel.php

```diff
@@ -156,7 +156,7 @@ protected function getListQuery()
 
 		// Join over the language
 		$query->select('l.title AS language_title, l.image AS language_image')
-			->join('LEFT', $db->quoteName('#__languages') . ' AS l ON l.lang_code = a.language');
+		->join('LEFT', $db->quoteName('#__languages') . ' AS l ON l.lang_code = a.language');
```

Comment:
```suggestion
			->join('LEFT', $db->quoteName('#__languages') . ' AS l ON l.lang_code = a.language');
```
Since this line belongs to the previous line, i.e. they are one statement, this line needs to be intended by one tab more than the previous one.